src/Workspaces/CSharp/Portable/Formatting/Rules/SpacingFormattingRule.cs

```diff
@@ -102,7 +103,7 @@ public override AdjustSpacesOperation GetAdjustSpacesOperation(SyntaxToken previ
             if (currentKind == SyntaxKind.CloseParenToken &&
                 (currentParentKind == SyntaxKind.IfStatement || currentParentKind == SyntaxKind.WhileStatement || currentParentKind == SyntaxKind.SwitchStatement ||
                 currentParentKind == SyntaxKind.ForStatement || currentParentKind == SyntaxKind.ForEachStatement || currentParentKind == SyntaxKind.DoStatement ||
-                previousParentKind == SyntaxKind.CatchDeclaration))
+                previousParentKind == SyntaxKind.CatchDeclaration || currentParentKind == SyntaxKind.UsingStatement))
```

Comment:
Sure. Were you thinking simply reorder the last two, so all currentParentKinds are before the previousParentKind?

Or more like this with previousParentKind broken out to a new line
```
if (currentKind == SyntaxKind.CloseParenToken &&
    (currentParentKind == SyntaxKind.IfStatement || currentParentKind == SyntaxKind.WhileStatement ||  currentParentKind == SyntaxKind.SwitchStatement ||
    currentParentKind == SyntaxKind.ForStatement || currentParentKind == SyntaxKind.ForEachStatement || currentParentKind == SyntaxKind.DoStatement ||
    currentParentKind == SyntaxKind.UsingStatement ||
    previousParentKind == SyntaxKind.CatchDeclaration))
```